e2e/aws/fixtures_test.go

```diff
@@ -208,11 +218,35 @@ func withDatabaseService(t *testing.T, matchers ...services.ResourceMatcher) tes
 func withFullDatabaseAccessUserRole(t *testing.T) testOptionsFunc {
 	t.Helper()
 	// Create a new role with full access to all databases.
-	return withUserRole(t, "db-access", types.RoleSpecV6{
-		Allow: types.RoleConditions{
-			DatabaseLabels: types.Labels{types.Wildcard: []string{types.Wildcard}},
-			DatabaseUsers:  []string{types.Wildcard},
-			DatabaseNames:  []string{types.Wildcard},
-		},
-	})
+	return withUserRole(t, hostUser, "db-access", allowDatabaseAccessRoleSpec)
+}
+
+var allowDatabaseAccessRoleSpec = types.RoleSpecV6{
+	Allow: types.RoleConditions{
+		DatabaseLabels: types.Labels{types.Wildcard: []string{types.Wildcard}},
+		DatabaseUsers:  []string{types.Wildcard},
+		DatabaseNames:  []string{types.Wildcard},
+	},
+}
+
+var autoDBUserKeepSpec = types.RoleSpecV6{
+	Allow: types.RoleConditions{
+		DatabaseLabels: types.Labels{types.Wildcard: []string{types.Wildcard}},
+		DatabaseUsers:  []string{types.Wildcard},
```

Comment:
```suggestion
		DatabaseRoles:  []string{"role1", "role2"},
```
DB users are not required. Also, in normal usage, at least db roles should be specified. Could we update test to provision some roles and check if the auto-provisioned user gets those roles assigned?

Reply:
> DB users are not required

ah ok, thanks!

And i'll update to add a couple roles

Reply:
updated to provision two roles for mysql and postgres, and then run a test that exercises the combined permissions of those roles to assert that both roles are auto granted.